README.md

```diff
@@ -1,8 +1,9 @@
 <p align="center"> <img width="250" height="250" src="opt_out_logo.png"> </p>
 
-<p></p>
-<h1 align="center"> Opt Out Tools </h1>
-<h2 align="center"> We are building tools to help female-identifying people engage with healthy online discussion </h2>
+<p></p> <h1 align="center"> Opt Out Tools </h1>
+
+<h2 align="center"> We are building tools to help female-identifying people
+engage with healthy online discussion avoiding misogyny </h2>
```

Comment:
We are building misogyny removing tools, to help female-identifying people engage with healthy online discussion.